syntax = "proto3";
message ProtocolBuffer {
    string collection_id; 
    string episode_id;
    string episode_url;
}